packages/auth-construct/README.md

```diff
@@ -0,0 +1,135 @@
+# AmplifyAuth Construct
+
+This package vends an L3 CDK Construct wrapping the behavior of Amplify Auth. This enables quick development and interation with AWS Cognito.
+
+The primary way to use this construct is to invoke it with a configuration object. You can declare the individual settings for your authentication by passing them as properties to the AmplifyAuth construct.
+
+Note: only a single instance of the `AmplifyAuth` construct can be invoked within a CDK synthesis at this point in time.
+
+## Examples
+
+### Simple email login with default settings
+
+In this example, we create a simple stack with email login enabled (by default). Deploying this will create a UserPool, UserPoolClient, IdentityPool, and Authenticated/Unauthenticated Roles.
+
+We create a cdk App and Stack for this example. You may be deploying this to a custom stack, but this is purely illustrative for a concise demo.
+
+```ts
+import { App, Stack } from 'aws-cdk-lib';
+import { AmplifyAuth } from '@aws-amplify/auth-construct';
+
+const app = new App();
+const stack = new Stack(app, 'Auth');
+
+new AmplifyAuth(stack, 'Auth');
+```
```

Comment:
could you create an issue in GH to find a solution to this problem: "How do we keep these samples in sync?".
One of the options is to somehow keep them in TS files to keep them under compiler eye and have some automation that detect diff between MD file and snippets.

Reply:
I have the same question regarding all of docs

Reply:
docs are going to be harder problem to solve outside of the repo, but if we build mechanism here we could then spread it to docs.